asf_search/download/download.py

```diff
@@ -74,14 +73,21 @@ def download_url(url: str, path: str, filename: str = None, session: ASFSession
         for chunk in response.iter_content(chunk_size=8192):
             f.write(chunk)
 
-def remotezip(url: str, session: ASFSession) -> RemoteZip:
+def remotezip(url: str, session: ASFSession):
```

Comment:
```suggestion
def remotezip(url: str, session: ASFSession) -> 'RemoteZip':
```
Using stings for type declaration allow for a "forward declaration"


If you decide to stick with warning instead of raising, you'll want to do:
```python
def remotezip(url: str, session: ASFSession) -> Optional['RemoteZip']:
```